.gitlab/lassen.yml

```diff
@@ -29,5 +29,5 @@
 
 opt_mpi_cuda_xl_16_1_1_8:
   variables:
-    SPEC: "%xl@16.1.1.8 +mpi +cuda cuda_arch=sm_70"
+    SPEC: "%xl@16.1.1.8 +mpi +cuda cuda_arch=70"
```

Comment:
@v-dobrev can you take a quick look?

Reply:
Do we have to change the `cuda_arch` checks in the mfem spack package for that to work? Or have they recently changed? I've seen other packages use just the number as  well.

Reply:
@jandrej, this is a change in the CI because the logic in the Spack package changed.

The logic changed because MFEM package class now inherits from `CudaPackage` which defines the `cuda_arch` variant and the `cuda_arch_values`.

This is consistent with other packages also inheriting from the CudaPackage class like Serac, RAJA and such.